### PR TITLE
{bio}[foss/2018b] freebayes v1.3.2

### PR DESCRIPTION
**Pull request by SethosII**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by SethosII** [comment]
Test report by @SethosII
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
ribnode015 - Linux ubuntu 18.04, x86_64, Intel(R) Xeon(R) CPU           E5645  @ 2.40GHz, Python 2.7.17
See https://gist.github.com/0a8d678baf1a0169c5463fdbc5b09d64 for a full test report.

File: easybuild/easyconfigs/f/freebayes/freebayes-1.3.2-foss-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+# To download freebayes, use git with the --recursive option.
+# This is required in order to download all nested git submodules for external repositories.
+# Afterwards pack everything into a tarball:
+#   git clone --recursive -b v1.3.2 git://github.com/ekg/freebayes.git
+#   tar czf freebayes-1.3.2.tar.gz freebayes
+# no checksum for freebayes-1.3.2.tar.gz since it's created from git repo
+sources = ['%(name)s-%(version)s.tar.gz']
```
Comment (verdurin) [inline easybuild/easyconfigs/f/freebayes/freebayes-1.3.2-foss-2018b.eb:19]: @SethosII to work around the lack of checksums, please add:
`checksums = [None]`

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/f/freebayes/freebayes-1.3.2-foss-2018b.eb (hunk 1 above)
Comment (SethosII) [reply easybuild/easyconfigs/f/freebayes/freebayes-1.3.2-foss-2018b.eb:19]: @verdurin I changed it.

Comment (boegel) [reply easybuild/easyconfigs/f/freebayes/freebayes-1.3.2-foss-2018b.eb:19]: @SethosII A better option would be to let EasyBuild download the sources via `git clone --recursive`, see https://docs.easybuild.io/en/latest/Writing_easyconfig_files.html#common-easyconfig-param-sources-git-config

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/f/freebayes/freebayes-1.3.2-foss-2018b.eb (hunk 1 above)
Comment (SethosII) [reply easybuild/easyconfigs/f/freebayes/freebayes-1.3.2-foss-2018b.eb]: @boegel Thanks for the hint, I changed it.

**Comment by SethosII** [comment]
Test report by @SethosII
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
ribnode015 - Linux ubuntu 18.04, x86_64, Intel(R) Xeon(R) CPU           E5645  @ 2.40GHz, Python 2.7.17
See https://gist.github.com/ca10f256ef5cd591b15084f46f92e9a4 for a full test report.

**Comment by SethosII** [comment]
Test report by @SethosII
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
ribnode015 - Linux ubuntu 18.04, x86_64, Intel(R) Xeon(R) CPU           E5645  @ 2.40GHz, Python 2.7.17
See https://gist.github.com/7c27727a556b1e5bbd720662bde52503 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@SethosII Lots of `samtools: command not found` errors in the failing tests, so missing dependency on `SAMtools`?

How did this work for you though, do you have a system-wide installation of `samtools`?

**Comment by SethosII** [comment]
@boegel Uh, `SAMtools` was installed on *some* machines. I added it and also added some more dependencies. They changed their build system in a recently released version and there was a clean definition of dependencies: https://github.com/freebayes/freebayes/blob/master/guix.scm.

**Comment by SethosII** [comment]
Test report by @SethosII
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
ribnode015 - Linux ubuntu 18.04, x86_64, Intel(R) Xeon(R) CPU           E5645  @ 2.40GHz, Python 2.7.17
See https://gist.github.com/c59a9e22027b5cbdaa7e5c30025cb459 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 2 out of 3 (1 easyconfigs in total)
vera-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/82a7a8d60348d44fa0443f7dfe9e57f4 for a full test report.

**Comment by boegel** [comment]
closing this, since foss/2018b is now a deprecated toolchain (see https://docs.easybuild.io/en/latest/Deprecated-easyconfigs.html#deprecated-toolchains) and we have easyconfigs for more recent versions already (see #15218)

**Pull request closed without merging**